### ISSUE TITLE
use a full map by default

### ISSUE THREAD

**Issue by ohnorobo** [codeforboston/finda#8]
MAPC tiles are very cool looking, but they only cover Mass. 
A sensible default would be to use the old Mapquest tiles (or something more neutral if you can find it), and have a config option to change the map/attribution. 

The gh-pages branch should be configured to use the current MAPC tiles.

**Comment by beechnut** [comment]
I know of a tileset on my other machine. Will explore.

**Comment by beechnut** [comment]
Similar in style to the MAPC basemap, but it covers the country.

``` javascript

L.tileLayer('http://{s}.tile.cloudmade.com/{key}/{styleId}/256/{z}/{x}/{y}.png', {
    key: 'd4fc77ea4a63471cab2423e66626cbb6',
    attribution: '&copy; <a href="http://openstreetmap.org/copyright">OpenStreetMap</a> contributors',
    styleId: 22677
  }).addTo(map);

```

I think I found this tileset through a Leaflet tutorial. I'm not sure if it's okay to use this key, though.

**Comment by ohnorobo** [comment]
The key is the one leaflet uses in their [public repo](https://github.com/Leaflet/Leaflet/blob/master/debug/vector/vector-mobile.html) so it's probably not okay to use on an active site. Should be alright for testing though. I'd rather use something unkeyed, but that probably limits our options a lot.

**Comment by paulswartz** [comment]
Other tile sources I came across on the OSM wiki:
- http://openmapsurfer.uni-hd.de/contact.html
- http://developer.mapquest.com/web/products/open/map
- http://www.openstreetmap.org/
- https://github.com/Esri/esri-leaflet (for ESRI base maps)

The MapQuest ones are a little cleaner; the uni-hd ones display more data.

**Comment by paulswartz** [comment]
Closed by #31.
